### PR TITLE
WIP PR bot

### PR DESCRIPTION
**Pull request by olamothe**

### CODE REVIEW

File: .github/workflows/prbot.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+jobs:
+  run-script:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+          node-version: '12'
```
Comment (samisayegh) [inline .github/workflows/prbot.yml:13]: Could we use node 14 or 15 here?

**Comment by olamothe** [comment]
This is a test comment

File: .gitignore [hunk 2]
```diff
@@ -5,5 +5,4 @@ npm-debug.log
 .ds_store
 dist
 veracode
-.size-snapshot.json
-reports
```
Comment (samisayegh) [inline .gitignore:9]: Hmm, removing reports from `.gitignore` might unintentionally bring the `eslint.xml` into version control when running `npm run lint:junit`. Should we exclude `*eslint.xml` instead?

File: scripts/reports/github-client.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+const {Octokit} = require('@octokit/rest');
+const credentials =
+  process.env.GITHUB_CREDENTIALS || '82ecb83d237039ee70cfbad6d77b2666cd52aed9';
```
Comment (samisayegh) [inline scripts/reports/github-client.js:3]: Should this key be checked into version control?

File: scripts/reports/pr-report-github.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+const {newClient} = require('./github-client');
+
+async function main() {
+  const c = newClient();
+  const issues = await c.issues.listForRepo({repo : 'ui-kit', owner: 'coveo', state: 'open'})
+  console.log(issues)
```
Comment (samisayegh) [inline scripts/reports/pr-report-github.js:6]: What does this bot do? Is it just retrieving open issues and logging them to the console? Why is this helpful?

**Comment by samisayegh** [comment]
Edit: I see now this is a WIP and should probably have not reviewed

**Comment by olamothe** [comment]
Pull Request Report

PR Title

  Title should follow the conventional commit spec:

<type>(optional scope): <description>

Example:

feat(headless): add result-list controller

**Comment by olamothe** [comment]
**Pull Request Report**

  
  **PR Title**

  
  :x: Title should follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) spec:
  
  <type>(optional scope): <description>

  Example:
  
  feat(headless): add result-list controller
  
  

  BUNDLE_SIZE_TODO

**Comment by olamothe** [comment]
**Pull Request Report**

  
  **PR Title**

  
  :x: Title should follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) spec:
  
  <type>(optional scope): <description>

  Example:
  
  feat(headless): add result-list controller

**Pull request closed without merging**